### ISSUE TITLE
ofDrawArrow looks like ass

### ISSUE THREAD

**Issue by golanlevin** [openframeworks/openFrameworks#2859]
With respect, ofDrawArrow has a pretty embarrassing problem. The cone (arrowhead) is not co-axial with the vector of the arrow itself. It seems like an XY coordinate flip error. For example: 

![ofdrawarrow](https://f.cloud.github.com/assets/290053/2428894/e25acd8a-ac60-11e3-8d47-67a47f0c677f.png)

That was produced with the code: 

```
    ofSetColor(0,0,0);
    ofVec3f arrowTailPoint = ofVec3f(0,0,0);
    ofVec3f arrowHeadPoint = ofVec3f(150,100,0);
    ofDrawArrow(arrowTailPoint, arrowHeadPoint, 20.0);
```

Much less important, a small aesthetic detail is that the arrowhead cone would look better if it were longer than it is wide, (say) in the ratio 2:1.

**Comment by aceslowman** [comment]
Is this problem present once again in 0.10? It doesn't appear to be applying any rotation to the cone.

![screenshot 2018-11-03 18 04 09](https://user-images.githubusercontent.com/6826702/47958573-4d9b2580-df93-11e8-8bed-2b8258a3fbbc.png)

**Comment by tgfrerer** [comment]
I believe this has been fixed with #6164 - did you try using the -patch-release branch of openframeworks? 

With compliments to @golanlevin for the idiosyncratic issue headline :)
